hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/validation/ExpiryMeta.java

```diff
@@ -24,10 +26,9 @@
  *
  * @param expiry the consensus second at which the entity expires
  * @param autoRenewPeriod the number of seconds between auto-renewals
- * @param autoRenewNum the number of the account to be charged for auto-renewals
+ * @param autoRenewAccountId the id of the account to be charged for auto-renewals
  */
-public record ExpiryMeta(
-        long expiry, long autoRenewPeriod, long autoRenewShard, long autoRenewRealm, long autoRenewNum) {
+public record ExpiryMeta(long expiry, long autoRenewPeriod, AccountID autoRenewAccountId) {
```

Comment:
Where is `ExpiryMeta` stored? Is it in state? If not, on restart, how is it recreated? If it is in state, should it be defined in protobuf?

Reply:
This is record just used for convenience to validate auto renew accountId , auto renew period in `ExpiryValidator`.
This is created from the data available from an entity in state during update transaction or from the transaction body during create transaction.